pom.xml

```diff
@@ -78,20 +78,20 @@
 
     <!-- JAR versions -->
     <version.assertj>3.5.2</version.assertj>
```

Comment:
Starting with `3.6.0`, breaking changes are introduced. Should be addressed separately.